enterprise/internal/insights/resolvers/insight_series_resolver.go

```
@@ -69,7 +69,10 @@ func unwrapSearchContexts(ctx context.Context, loader SearchContextLoader, rawCo
 				return nil, nil, errors.Wrapf(err, "failed to parse search query for search context: %s", rawContext)
 			}
 			inc, exc := plan.ToQ().Repositories()
-			include = append(include, inc...)
+			for _, repoFilter := range inc {
+				repo, _ := search.ParseRepositoryRevisions(repoFilter)
```

could you add a comment to say what that we're ignoring revision specifiers here (and maybe why we don't need them?) at first I thought this might be ignoring an error

---

👍 will add a comment here explaining the behavior